src/data-mgt/node/utils/transform.js

```diff
@@ -0,0 +1,302 @@
+const axios = require("axios");
+const isEmpty = require("is-empty");
+const { logElement, logText, logObject } = require("./log");
+const constants = require("../config/constants");
```

Comment:
merged all utils for measurement transformation into one file. As I prepare for the integration of unit tests to all files in the codebase.